views/journal/talkform.tt

```diff
@@ -219,98 +219,83 @@ the same terms as Perl itself.  For a copy of the license, please reference
+
+            <label for='talkpostfromlj'>
+              [% dw.img( 'id_user', '' ) %]
+              [% IF remote && ! remote.openid_identity %] Other [%- END %]
+              [% '/talkpost.bml.opt.siteuser' | ml( sitename => site.nameshort ) %]
+            </label>
             [%- IF journal.screens_all -%]
               [%- '/talkpost.bml.opt.willscreen' | ml -%]
             [%- ELSIF journal.screens_non_access -%]
               [%- '/talkpost.bml.opt.willscreenfriend' | ml -%]
             [%- END -%]
-          </td>
-        </tr>
+        </div>
 
         [%# site user login form, always present but sometimes hidden -%]
-        <tr valign='middle' align='left' id='lj_more'>
-          <td colspan='2'></td>
-
-          <td>
-            <table summary=''>
-              <tr>
-                <td>
-                  [%- 'Username' | ml -%][%# NF: ??? -%]
-                </td>
-                <td>
-                  [%- form.textbox(
-                    name = 'userpost'
-                    id = 'username'
-                    size = 13
-                    maxlength = 25
-                    value = ( comment.user || remote.user )
-                    onclick = 'this.value=""'
-                    style = "background: url('${site.imgroot}/silk/identity/user.png') no-repeat; background-color: #fff; background-position: 0px 1px; padding-left: 18px; color: #00C; font-weight: bold;"
-                  ) -%]
-                </td>
-              </tr>
-
-              <tr>
-                <td>
-                  [%- 'Password' | ml -%][%# NF: ??? -%]
-                </td>
-                <td>
-                  <input class='textbox' name='password' type='password' maxlength='30' size='18' id='password' />
-                </td>
-              </tr>
-
-              <tr>
-                <td colspan='2'>
-                  <label for='logincheck'>[% '/talkpost.bml.loginq' | ml %]&nbsp;</label>
-                  <input type='checkbox' name='do_login' id='logincheck' />
-                </td>
-              </tr>
-            </table>
-          </td>
-        </tr>
+        <div class="from-login" id="from-user-more">
+
+          <div>
+            [%- form.textbox(
+              name = 'userpost'
+              id = 'username'
+              label = dw.ml('Username')
+              size = 13
+              maxlength = 25
+              value = ( comment.user || remote.user )
+              'data-revert-value' = ( comment.user || remote.user )
+              style = "background: url('${site.imgroot}/silk/identity/user.png') no-repeat; background-color: #fff; background-position: 0px 1px; padding-left: 18px; color: #00C; font-weight: bold;"
+            ) -%]
+          </div>
+
+          <div>
+            [%- form.password(
+              name = 'password'
+              id= 'password'
+              label = dw.ml('Password')
+              maxlength = 30
```

Comment:
Does this have a constant too?

Reply:
Doesn't appear to. 30 is what's used in most places, but I do think I saw a 50 somewhere random.

Reply:
I don't believe it has a constant, no.